### PR TITLE
Py3stage1

### PR DESCRIPTION
**Pull request by liewhnic**
Stage 1 changes applied. Flake8 compliance checked. 

I ran:
```bash
futurize -w  --stage1 setup.py __init__.py pyrovito pyrovito_utils.py
```
but in my case, futurize still complains about `pyrovito`.

### CODE REVIEW

**Comment by degv364** [comment]
@jcrono  can correct me if I am wrong, but I think that should also futurize setup.py.

Also can you explain better how futurize is complaining about pyrovito?

File: __init__.py [hunk 1]
```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env python
```
Comment (degv364) [inline __init__.py:1]: Please also add encoding to all your files:

"# -*- coding: utf-8 -*-"

**Comment by liewhnic** [comment]
@degv364, when I run:

```bash
futurize -w --stage1 pyrovito
```
at the source directory, console gives next message:

```bash
RefactoringTool: No changes to pyrovito
RefactoringTool: Files that need to be modified:
RefactoringTool: pyrovito
```
Is not big deal. But considering futurize changes were already written, this seems strange to me.

**Comment by jprltsnz** [comment]
> @jcrono can correct me if I am wrong, but I think that should also futurize setup.py.

We *could* import the futurize functions, but the setup.py is already python2 and python3 compatible, so there is no need.

File: __init__.py (hunk 1 above)
Comment (jprltsnz) [reply __init__.py:1]: ditto but rember to escape the `*` or use a codeblock  because of markdown @degv364 : ` # -*- coding: utf-8 -*-`

**Review by jprltsnz — CHANGES_REQUESTED** [review]

**Comment by liewhnic** [comment]
Encoding indication added.

**Review by degv364 — APPROVED** [review]

**Review by jprltsnz — APPROVED** [review]

**Pull request merged**